package.json

```diff
@@ -13,12 +13,18 @@
     "lint": "./node_modules/eslint/bin/eslint.js src/"
   },
   "jest": {
-    "scriptPreprocessor": "./preprocessor.js",
+    "rootDir": "./src",
+    "collectCoverage": true,
+    "scriptPreprocessor": "../preprocessor.js",
     "unmockedModulePathPatterns": [
       "./node_modules/"
     ],
     "testPathIgnorePatterns": [
       "./node_modules/"
+    ],
+    "testFileExtensions": [
+      "js",
+      "jsx"
```

Comment:
@jackdcrawford if you modify `testFileExtensions` to have just `js` (or remove it entirely) it'll run coverage, but only show [coverage for a single file](https://coveralls.io/builds/2548358). If you run it with `jsx`, you get [coverage for everything](https://coveralls.io/builds/2548696) but Button and Flag tests fail.